tasks/security_agent.py

```diff
@@ -253,14 +253,18 @@ def ninja_c_syscall_tester_common(nw, file_name, build_dir, flags=None, libs=Non
     if static:
         flags.append("-static")
 
+    additional_flags = [f"-isystem/usr/include/{uname_m}-linux-gnu"]
+    if compiler == "clang":
+        additional_flags.append(f"-D__{uname_m}__")
```

Comment:
why is this required only for clang ?

Reply:
yeah from what I tested you can actually remove completely this define in and it should work perfectly fine

Reply:
That's good to know. Thanks for having tested it !